packages/legend-graph/src/graph/metamodel/pure/packageableElements/service/Service.ts

```diff
@@ -65,6 +65,8 @@ export class Service extends PackageableElement implements Hashable, Testable {
       this.test ?? '',
       hashArray(this.tests),
       hashArray(this.postValidations),
+      hashArray(this.taggedValues),
+      hashArray(this.stereotypes.map((val) => val.pointerHashCode)),
```

Comment:
why do we use `.pointerHashCode` in some places. what is correct here.

Reply:
<img width="384" alt="Screenshot 2024-08-26 at 1 36 31 PM" src="https://github.com/user-attachments/assets/95fdf86d-0781-4991-8def-c0fc733e71f1">
I think this is how m3-stereotypes is calculating hashcode